src/types/http.d.ts

```diff
@@ -29,11 +30,13 @@ export declare class Http {
     oauth: OAuth,
     cacheStore?: CacheStorage,
     cacheMiddleware?: typeof defaultCacheMiddleware | unknown,
+    httpsProxy?: string | HttpsProxyAgentOptions,
   });
   defaultHeaders: Record<string, unknown>;
   requestExecutor: RequestExecutor;
   cacheStore: CacheStorage;
   cacheMiddleware: typeof defaultCacheMiddleware | unknown;
+  agent: typeof HttpsProxyAgent | unknown;
```

Comment:
Use `any` here, so `allowSyntheticDefaultImports` won't be needed for types test. The concern for `allowSyntheticDefaultImports ` is it may break downstream types by introducing this loose flag.